static/docs/src/pages.yaml

```diff
@@ -114,8 +138,24 @@ definitions:
       summary:
         type: string
         description: User provided summary of the page.  No more than 140 characters.
+      versionId:
+        $ref: 'pageversions.yaml#/definitions/pageVersionId'
       pageTemplateId:
         $ref: 'pagetemplates.yaml#/definitions/pageTemplateId'
+      permissionType:
+        $ref: '#/definitions/permissionType'
+  'permissionType':
+    type: string
+    enum:
+    - PR
+    - PU
+    - PO
+    - LO
+    description: |
+      * **PR**: Private. Only the owner(s) may edit/see the page.
+      * **PU**: Public.  Everyone may edit/see the page and the page is searchable.
+      * **PO**: Public Only. Everyone may see the page and the page is searchable.
```

Comment:
When would you do `PO` over `PU`?

Reply:
PO: If you want to make something publicly viewable that only you want edit access to
PU: If you want to make something that anyone can modify, like a wiki page